site/en/examples/chat_calculator.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://developers.generativeai.google/examples/chat_calculator\"><img src=\"https://developers.generativeai.google/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on Generative AI</a>\n",
+        "    <a target=\"_blank\" href=\"https://ai.google.dev/examples/chat_calculator.ipynb\"><img src=\"https://developers.generativeai.google/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on Generative AI</a>\n",
```

Comment:
Is the site-link meant to include *.ipynb? Will that still download the notebook?

Reply:
Fixed.